lib/goth/server.ex

```diff
@@ -5,6 +5,7 @@ defmodule Goth.Server do
   alias Goth.Token
 
   @max_retries 3
+  @registry GothRegistry
```

Comment:
```suggestion
  @registry Goth.Registry
```

Reply:
I have doubts about it but not preference at all, so let's change it

Reply:
What are your doubts?

Reply:
About syntax nothing else use only one word or a namespaced one, nothing more than that